src/app/components/icons/IconDrawer.tsx

```diff
@@ -150,16 +172,82 @@ export const IconDrawer: React.FC = () => {
+                                    <InputLabel id="icon-color-select-label">Select a Color:</InputLabel>
+                                    {selectedIcon.isMaterial ? (
+                                        <Select
+                                            labelId="icon-color-select-label"
+                                            id="icon-color-select"
+                                            value={iconColor}
+                                            onChange={handleColorChange}
+                                        >
+                                            <MenuItem value={'Black'}>Black</MenuItem>
+                                            <MenuItem value={'White'}>White</MenuItem>
+                                        </Select>
+                                    ) : (
+                                        <Select
+                                            labelId="icon-color-select-label"
+                                            id="icon-color-select"
+                                            value={iconColor}
+                                            onChange={handleColorChange}
+                                        >
+                                            <MenuItem value={'Black'}>Black</MenuItem>
+                                            <MenuItem value={'White'}>White</MenuItem>
+                                            <MenuItem value={'Blue'}>Blue</MenuItem>
+                                            <MenuItem value={'Gray'}>Gray</MenuItem>
+                                        </Select>
+                                    )}
+                                </FormControl>
+                            </div>
                             <div>
                                 <Button
                                     variant="contained"
                                     color="primary"
                                     style={{ marginRight: theme.spacing(1) }}
                                     startIcon={<GetApp />}
-                                    onClick={(): void => downloadSvg(selectedIcon)}
+                                    onClick={(): void => {
+                                        void downloadSvg(selectedIcon, iconColor, iconSize);
+                                    }}
                                 >
                                     SVG
                                 </Button>
+                                <Button
+                                    variant="contained"
+                                    color="primary"
+                                    onClick={(): void => downloadPng(selectedIcon, iconColor, iconSize)}
+                                    startIcon={<GetApp />}
+                                >
+                                    PNG
+                                </Button>
                                 {!selectedIcon.isMaterial && (
```

Comment:
You can remove this text block about opening in a new window now as it is no longer applicable